src/main/java/uk/gov/hmcts/dts/fact/services/admin/AdminRole.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.hmcts.dts.fact.services.admin;
+
+public final class AdminRole {
+    public static final String FACT_SUPER_ADMIN = "fact-super-admin";
+    public static final String FACT_ADMIN = "fact-admin";
+
+    private AdminRole() {
```

Comment:
I thought I saw this one in another PR? :O I'm prob wrong. Either way, did the no args constructor annotation work for this one? No worries if not

Reply:
No I tried it didn't work...can't remember what is the issue now